src/main/java/com/resource/management/api/Resource.java

```diff
@@ -25,4 +26,6 @@
     private List<String> crew;
 
     private ResourceStatus status;
+
+    private ResourceType type;
```

Comment:
This does not make sense for equipment. We need totally different fields(and my impression, a different type) for this.

A saw will not have a vehicle type and a plate number, etc.